sse/src/main/scala/io/finch/sse/ServerSentEvent.scala

```diff
@@ -0,0 +1,51 @@
+package io.finch.sse
+
+import java.nio.charset.Charset
+
+import cats.Show
+import com.twitter.concurrent.AsyncStream
+import com.twitter.io.Buf
+import io.finch.{Encode, Ok, Output, Text}
+import io.finch.internal.{BufText, LowPriorityToResponseInstances}
+
+case class ServerSentEvent[A](
+  data: A,
+  id: Option[String] = None,
+  event: Option[String] = None,
+  retry: Option[Long] = None
+)
+
+object ServerSentEvent extends LowPriorityToResponseInstances {
+  type EventStream[A] = AsyncStream[ServerSentEvent[A]]
+
+  def stream[A](sse: EventStream[A]): Output[EventStream[A]] = {
+    Ok(sse).withHeader("Cache-Control" -> "no-cache").withHeader("Connection" -> "keep-alive")
```

Comment:
Can we move that part into a cookbook? I'm not sure we need an explicit API method for this.